### DISCUSSION TITLE
一分钟读论文：《影响大规模敏捷软件开发按时交付的因素》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#6] [Announcements]
# 一分钟读论文：《影响大规模敏捷软件开发按时交付的因素》

荷兰代尔夫特理工大学与荷兰国际集团 (ING) 合著的论文《Factors Affecting On-Time Delivery in Large-Scale Agile Software Development》 对来自 ING 的 635 名工程师进行调查，分析了 185 个团队和 2,208 个 Epic 数据集。发现： 需求细化、任务依赖性、组织一致性、组织政治和团队的地理分布被认为对及时的软件交付有最大的影响。 项目规模、依赖项数量、历史交付绩效、团队熟悉度和开发人员经验是解释软件交付进度偏差的最重要变量。 因素按层次相互作用：组织因素被认为与人员因素相互作用，进而影响技术因素。技术因素被认为对及时的软件交付有直接影响。

https://unbug.github.io/Factors-Affecting-On-Time-Delivery-in-Large-Scale-Agile-Software-Development/

**Comment by unbug** [comment]
我很意外但不能同意更多：组织政治被认为对及时的软件交付有最大的影响
